Flaky test Apache CXF testMultipleServerRequests

CI: https://travis-ci.org/opentracing-contrib/java-jaxrs/builds/524893506
```
testMultipleServerRequests(io.opentracing.contrib.jaxrs2.itest.cxf.ApacheCXFITest)  Time elapsed: 0.352 sec  <<< FAILURE!
java.lang.AssertionError: expected:<100> but was:<99>
	at org.junit.Assert.fail(Assert.java:88)
	at org.junit.Assert.failNotEquals(Assert.java:834)
	at org.junit.Assert.assertEquals(Assert.java:645)
	at org.junit.Assert.assertEquals(Assert.java:631)
	at 


Results :
Failed tests:   testMultipleServerRequests(io.opentracing.contrib.jaxrs2.itest.cxf.ApacheCXFITest): expected:<100> but was:<99>
Tests run: 42, Failures: 1, Errors: 0, Skipped: 0
```

--- a/opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java
+++ b/opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java
@@ -315,6 +315,7 @@ public abstract class AbstractServerTest extends AbstractJettyTest {
 
         executorService.shutdown();
         executorService.awaitTermination(1, TimeUnit.SECONDS);
+        await().until(finishedSpansSizeEquals(numberOfThreads*10));
 
         List<MockSpan> mockSpans = mockTracer.finishedSpans();
         assertOnErrors(mockSpans);

--- a/opentracing-jaxrs2-itest/pom.xml
+++ b/opentracing-jaxrs2-itest/pom.xml
@@ -19,9 +19,9 @@
   </modules>
 
   <properties>
-    <version.org.apache.cxf>3.2.6</version.org.apache.cxf>
-    <version.org.eclipse.jetty>9.4.12.v20180830</version.org.eclipse.jetty>
-    <version.org.glassfish.jersey>2.27</version.org.glassfish.jersey>
+    <version.org.apache.cxf>3.3.1</version.org.apache.cxf>
+    <version.org.eclipse.jetty>9.4.17.v20190418</version.org.eclipse.jetty>
+    <version.org.glassfish.jersey>2.28</version.org.glassfish.jersey>
     <version.org.jboss.resteasy>3.6.1.Final</version.org.jboss.resteasy>
     <version.wildfly>13.0.0.Final</version.wildfly>
     <version.org.jboss.arquillian>1.4.0.Final</version.org.jboss.arquillian>

